packages/infra/lib/ihe-gateway-v2-stack.ts

```diff
@@ -259,4 +267,54 @@ export class IHEGatewayV2LambdasNestedStack extends NestedStack {
 
     return documentRetrievalLambda;
   }
+
+  private setupScheduledReportLambda(ownProps: {
+    lambdaLayers: LambdaLayers;
+    vpc: ec2.IVpc;
+    secrets: Secrets;
+    apiURL: string;
+    envType: EnvType;
+    sentryDsn: string | undefined;
+    dbCredsSecret: secret.ISecret;
+    dbCluster: rds.IDatabaseCluster;
+    dbReadOnlyEndpoint: rds.Endpoint;
+  }): Lambda {
+    const {
+      lambdaLayers,
+      vpc,
+      apiURL,
+      envType,
+      sentryDsn,
+      dbCredsSecret,
+      dbCluster,
+      dbReadOnlyEndpoint,
+    } = ownProps;
+
+    const scheduledReportLambda = createLambda({
+      stack: this,
+      name: "IHEGatewayV2ScheduledReport",
+      entry: "ihe-gateway-v2-scheduled-report",
+      envType: envType,
+      envVars: {
+        DB_CREDS: dbCredsSecret.secretArn,
+        API_URL: apiURL,
+        ...(sentryDsn ? { SENTRY_DSN: sentryDsn } : {}),
+        DB_READ_REPLICA_ENDPOINT: JSON.stringify(dbReadOnlyEndpoint),
+      },
+      layers: [lambdaLayers.shared],
+      memory: 512,
+      timeout: Duration.minutes(5),
+      vpc,
+    });
+
+    dbCluster.connections.allowDefaultPortFrom(scheduledReportLambda);
+    dbCredsSecret.grantRead(scheduledReportLambda);
+
+    const rule = new events.Rule(this, "ScheduledReportRule", {
+      schedule: events.Schedule.rate(Duration.hours(12)),
```

Comment:
Will this run every 12h from the time its deployed? If so, wouldn't it be better to have specific times on the day so we have more predictable Ops?

Reply:
I changed it to run at 12am and 2pm UTC (7am PST and 5pm PST